shell/platform/windows/external_texture_d3d.cc

```diff
@@ -14,18 +14,18 @@ ExternalTextureD3d::ExternalTextureD3d(
     const FlutterDesktopGpuSurfaceTextureCallback texture_callback,
     void* user_data,
     const AngleSurfaceManager* surface_manager,
-    const GlProcs& gl_procs)
```

Comment:
With this PR, do we still use the `GlProcs` struct anywhere? Should we keep the type in `external_texture.h` for now?

Reply:
Oops, good catch